app/src/main/java/com/hover/stax/utils/Utils.kt

```diff
@@ -147,9 +147,7 @@ object Utils {
     @JvmStatic
     fun formatAmountForUSSD(number: Double?): String {
         return try {
-            val formatter = DecimalFormat("###0.##")
-            formatter.maximumFractionDigits = 2
-            formatter.minimumFractionDigits = 0
+            val formatter = DecimalFormat("##0")
```

Comment:
This rounds up, but I think it's safe, cause all the times this has caused a failed transaction, the value was ```.00```

Reply:
This isn't right, some services do allow decimal values, so this wont work correctly for those.